.github/workflows/build-lint-test.yml

```diff
@@ -1,95 +1,40 @@
-name: Build, Lint, and Test
+name: Tests
 
 on:
   workflow_call:
 
 jobs:
-  prepare:
-    name: Prepare
+  build_and_test:
     runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js
-        uses: actions/setup-node@v3
-        with:
-          node-version-file: '.nvmrc'
-          cache: 'yarn'
-      - name: Install Yarn dependencies
-        run: yarn --immutable
 
-  build:
-    name: Build
-    runs-on: ubuntu-latest
-    needs:
-      - prepare
     strategy:
+      fail-fast: false
+      max-parallel: 4
       matrix:
-        node-version: [16.x, 18.x, 20.x]
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ matrix.node-version }}
-          cache: 'yarn'
-      - run: yarn --immutable --immutable-cache
-      - run: yarn build
-      - name: Require clean working directory
-        shell: bash
-        run: |
-          if ! git diff --exit-code; then
-            echo "Working tree dirty at end of job"
-            exit 1
-          fi
+        node-version: [18, 20, 21]
```

Comment:
How much do you really care about running this on three Node versions? :laughing:

Reply:
I don't really care at all lol. Copied these values from `js-stellar-sdk`, in lieu of metamask's `[16, 18, 20]`. The docs site we're using this on is always deployed using 20, so are we better off just testing on that one version?

Reply:
Yeah :p